spec/takeaway_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'takeaway'
+
+describe Takeaway do  
+
+	subject(:takeaway) {described_class.new(menu: menu, order: order)} #injecting menu class to allow amy menu methods to be handled in menu class (Order doesn't need to know!) - ## ?? ## Role of value menu?  Doesn't work without, but unsurr of role...
```

Comment:
Metrics/LineLength: Line is too long. [249/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)